magenta/models/nsynth/wavenet/fastgen.py

```diff
@@ -194,20 +213,39 @@ def synthesize(encodings,
             total_length,
         ), dtype=np.float32)
     audio = np.zeros([batch_size, 1])
-
+    current_time = time.time()
```

Comment:
Same here, I don't expect it to create a lot of overhead, but better to only do it if necessary.